TwitchPlaysAssembly/Src/Helpers/IRCConnection.cs

```diff
@@ -674,7 +674,7 @@ private void OutputThreadMethod(TextWriter output)
 			_state = IRCConnectionState.Disconnected;
 		}
 		if (!gameObject.activeInHierarchy)
-			AddTextToHoldable("[IRC:Disconnect] Twitch plays disabled");
+			AddTextToHoldable("[IRC:Disconnect] Twitch plays disabled.");
```

Comment:
Shouldn't it be Twitch Plays, not Twitch plays?

Reply:
Corrected in 21BC28F